addons/airway/stringtable.xml

```diff
@@ -70,7 +70,7 @@
         </Key>
         <Key ID="STR_KAT_Airway_SETTING_occluded_DESC">
             <English>Probability for an occlusion in percentage that gets rolled Setting:[Occlusion repeat timer] (default every 60s)</English>
-            <German>Atemwegsverlegungswahrscheinlichkeit durch Erbrochenes in prozent die durch Einstellung:[Zeit für die Wiederholung des Erbrechens] (standard alle 60s) berechnet wird.</German>
+            <German>Wahrscheinlichkeit für eine Okklusion in Prozent, die gewürfelt wird. Einstellung: [Timer für Okklusionswiederholung] (Standard alle 60 Sekunden)</German>
```

Comment:
> Wahrscheinlichkeit für eine Okklusion in Prozent, die gewürfelt wird. 
`Wahrscheinlichkeit für eine Okklusion in Prozent.` Sollte ausreichend sein, der Nebensatz ist nicht wirklich nötig.

@MiszczuZPolski the english orginial is a bit wierd, ngl. Whats the context here?